Automattic/wp-calypso
# Add/Login via the mobile app 2FA notice

#### Proposed Changes

This change adds a static notice message to the "Login via the mobile app" page to alert users that the feature is not available if they have enabled 2FA on their account. This new message needs translation.

![Screen Shot 2022-10-11 at 11 18 40 AM](https://user-images.githubusercontent.com/506707/195132957-c574fd1b-bb64-41b8-89d8-0bc69878f71c.png)


#### Testing Instructions
Navigate to the log in page
Click on "Login via the mobile app"
✅ Verify that the notify message "Login via the mobile app is not available if you've enabled two-step authentication on your account" is shown

#### Pre-merge Checklist
